website/docs/r/opc_database_service_instance.html.markdown

```diff
@@ -0,0 +1,168 @@
+  vm_public_key = "an ssh public key"
+  parameter {
+    admin_password = "Test_String7"
+    backup_destination = "NONE"
+    sid = "ORCL"
+    usable_storage = 15
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Service Instance.
+
+* `edition` - (Required) Database edition for the service instance. Possible values are SE, EE, EE_HP, or EE_EP.
+
+* `level` - (Required) Service level for the service instance. Possible values are BASIC or PAAS.
+
+* `shape` - (Required) Desired compute shape. Possible values are oc3, oc4, oc5, oc6, oc1m, oc2m, oc3m, or oc4m.
+
+* `subscription_type` - (Required) Billing unit. Possible values are HOURLY or MONTHLY.
+
+* `version` - (Required) Oracle Database software version; one of: 12.2.0.1, 12.1.0.2, or 11.2.0.4.
+
+* `vm_public_key` - (Required) Public key for the secure shell (SSH). This key will be used for authentication when connecting to the Database Cloud Service instance using an SSH client.
+
+* `parameter` - (Optional) Additional configuration for a service instance. This set is required if level is PAAS. Parameter is documented below.
+
+* `description` - (Optional) A description of the Service Instance.
+
+Parameter supports the following:
+
+* `admin_password` - (Required) Password for Oracle Database administrator users sys and system. The password must meet the following requirements: Starts with a letter. Is between 8 and 30 characters long. Contains letters, at least one number, and optionally, any number of these special characters: dollar sign ($), pound sign (#), and underscore ( _ )
+
+* `backup_destination` - (Required) Backup Destination. Possible values are BOTH, OSS, NONE.
+
+* `char_set` - (Required) Character Set for the Database Cloud Service Instance. Default value is AL32UTF8.
+
+`usable_storage` - (Required) Storage size for data (in GB). Minimum value is 15. Maximum value depends on the backup destination: if BOTH is specified, the maximum value is 1200; if OSS or NONE is specified, the maximum value is 2048.
+
+* `cloud_storage_container` - (Optional) Name of the Oracle Storage Cloud Service container used to provide storage for your service instance backups. Use the following format to specify the container name: <storageservicename>-<storageidentitydomain>/<containername>
+
+* `cloud_storage_username` - (Optional) Username for the Oracle Storage Cloud Service administrator.
+
+* `cloud_storage_password` - (Optional) Password for the Oracle Storage Cloud Service administrator.
+
+* `create_storage_container_if_missing` - (Optional) Specify if the given cloud_storage_container is to be created if it does not already exist. Valid values are true and false. Default value is false.
+
+* `disaster_recovery` - (Optional) Specify if an Oracle Data Guard configuration is created using the Disaster Recovery option or the High Availability option. Valid values are yes and no. Default value is no.
```

Comment:
Do this as true/false and then convert it to yes/no before we send it to the sdk?